tools/linter_lib/custom_check.py

```diff
@@ -278,18 +298,18 @@ def build_custom_checkers(by_lang):
     See WRAPPER_COMMENT in test_helpers.py for details.
     '''},
         # Directly fetching Message objects in e.g. views code is often a security bug.
-        {'pattern': '[^r][M]essage.objects.get',
+        {'pattern': '[^r]Message.objects.get',
```

Comment:
I'm pretty sure this one and the next one were written how they were to avoid firing on the linter code.

Reply:
I thought so as well, but the linter didn't complain. Will revert the respective commit anyways, though.

Reply:
I think it's reasonable to clean these up if the linter doesn't complain.  Sounds like we fixed that issue a smarter way :)